etc/talos/talos_pos_tracker.yaml

```diff
@@ -29,6 +29,8 @@ CONTROLLER:
                      "gripper_right_inner_single_joint",
                      "gripper_right_fingertip_3_joint",
                      "gripper_right_motor_single_joint"]
+  check_model_collisions: true
+  collision_path: /home/pal/inria_wbc/etc/talos/collisions/talos_collisions.yaml
```

Comment:
We should not use an absolute path here (like, it does not work on my computer).

Reply:
(we should use the base_path parameter)